bpfman/src/bin/cli/args.rs

```diff
@@ -307,9 +312,241 @@ pub(crate) struct GetArgs {
 
 #[derive(Subcommand, Debug)]
 #[command(disable_version_flag = true)]
+#[allow(clippy::large_enum_variant)]
 pub(crate) enum ImageSubCommand {
     /// Pull an eBPF bytecode image from a remote registry.
     Pull(PullBytecodeArgs),
+    /// Build an eBPF bytecode image from local bytecode objects, and push to a registry.
```

Comment:
I think this helptext needs to be expanded. With so many arches, it's hard to figure out how to use.

```suggestion
    /// Build an eBPF bytecode image from local bytecode objects and push to a registry.
    ///
    /// To use, the --container-file and --tag must be included, as well as the bytecode
    /// object file, which can be passed in several ways. Use either:
    /// * --bytecode: for a single image of host architecture.
    /// * --cilium-ebpf-project: to pass a path which contains multiple object files for
    ///   different architectures.
    /// * --bc-386-el .. --bc-s390x-eb: to add one or more architecture object files.
    ///
    /// Examples:
    ///    bpfman image build -f Containerfile.bytecode -t quay.io/<USER>/go-xdp-counter:test \
    ///      -b ./examples/go-xdp-counter/bpf_x86_bpfel.o
    #[clap(verbatim_doc_comment)]```

Reply:
Yeah sounds good... I was just doing my best to avoid adding sub commands i.e `bpfman image build multi-arch` etc, also making sure the args worked as expected was a PITA